sdk/python/kfp/_client.py

```diff
@@ -422,6 +422,13 @@ def schedule_pipeline(self, experiment_id, job_name, pipeline_package_path=None,
         params {dict} -- The pipeline parameters (default: {{}})
         pipeline_id {[type]} -- The id of the pipeline which should run on schedule (default: {None})
         namespace {[type]} -- The name space with which the pipeline should run (default: {None})
+        max_concurrency {[type]} -- Max number of concurrent runs scheduled (default: {10})
```

Comment:
Sorry we did not catch this before. It seems like `[type]` should refer to a concrete type. For example `max_concurrency {int}`

Reply:
Thanks for pointing out! Fixed